test/unit/CookiesStorageTests/cookieLoad_test.js

```diff
@@ -0,0 +1,39 @@
+goog.module('measurementLibrary.storage.CookiesStorage.testing.cookieLoad');
+goog.setTestOnly();
+
+const CookiesStorage = goog.require('measurementLibrary.storage.CookiesStorage');
+
+describe('The load method for cookiesStorage', () => {
+  const storage = new CookiesStorage({});
+
+  const addCookie = (key, value) => {
+    const expiry = new Date();
+    expiry.setMilliseconds(expiry.getMilliseconds() + 1000);
```

Comment:
Setting the expiration time and hoping that the code finishes in 1 second is a little sus. I think it works fine here, but perhaps add a comment explaining it since it's a bit weird?